src/Microsoft.Kusto.ServiceLayer/DataSource/IDataSource.cs

```diff
@@ -88,29 +86,6 @@ public interface IDataSource : IDisposable
         /// <param name="updateDatabase">Object metadata.</param>
         void UpdateDatabase(string databaseName);
 
-        /// <summary>
-        /// Gets autocomplete suggestions at given position.
-        /// </summary>
-        /// <param name="GetAutoCompleteSuggestions">Object metadata.</param>
-        CompletionItem[] GetAutoCompleteSuggestions(ScriptDocumentInfo queryText, Position index, bool throwOnError = false);
-        /// <summary>
-        /// Gets quick info hover tooltips for the current position.
-        /// </summary>
-        /// <param name="GetHoverHelp">Object metadata.</param>
-        Hover GetHoverHelp(ScriptDocumentInfo scriptDocumentInfo, Position textPosition, bool throwOnError = false);
-
-        /// <summary>
-        /// Gets definition for a selected query text.
-        /// </summary>
-        /// <param name="GetDefinition">Object metadata.</param>
-        DefinitionResult GetDefinition(string queryText, int index, int startLine, int startColumn, bool throwOnError = false);
-        
-        /// <summary>
-        /// Gets a list of semantic diagnostic marks for the provided script file
-        /// </summary>
-        /// <param name="GetSemanticMarkers">Object metadata.</param>
-        ScriptFileMarker[] GetSemanticMarkers(ScriptParseInfo parseInfo, ScriptFile scriptFile, string queryText);
-
```

Comment:
These 4 were moved to KustoIntellisenseHelper since they're related to that and didn't _need_ to be on KustoDataSource